WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Order Summary Section/SummaryTableViewCell/SummaryTableViewCellTests.swift

```diff
@@ -0,0 +1,121 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+final class SummaryTableViewCellTests: XCTestCase {
```

Comment:
nice thorough tests on the UI and action!

Reply:
Thanks but I simply copied and _fixed_ this from the original SummaryTableViewCellTests file. I moved that file so Github thought it's new. 😄